frontend/src/components/common/RadioSelectGroup.tsx

```diff
@@ -85,8 +113,13 @@ const RadioSelectGroup = (props: RadioSelectGroupProps) => {
 
   return (
     <FormControl isRequired={isRequired} m="2em 0">
-      <FormLabel>{label}</FormLabel>
-      {helperText && <FormHelperText mb="1em">{helperText}</FormHelperText>}
+      <FormLabel fontWeight="600">{label}</FormLabel>
+      <FormHelperText fontSize="16px" color="black" marginBottom="20px">
```

Comment:
not that I think black is going to change that much but could use `black.100` since the hex in the design system is actually #171717